src/settings_gui.cpp

```diff
@@ -39,14 +39,14 @@
 #include "rev.h"
 #include "video/video_driver.hpp"
 #include "music/music_driver.hpp"
+#include <gui.h>
```

Comment:
`<>` is for system includes, OpenTTD includes are done with `""`

Reply:
My bad--fixed!